get.sh

```diff
@@ -27,6 +27,11 @@ if ! echo $PATH | grep "/usr/local/bin" > /dev/null; then
   fi
 fi
 
+if ! $(grep "Defaults.*secure_path" /etc/sudoers | grep -q "/usr/local/bin"); then
```

Comment:
I don't feel comfortable about editing /etc/sudoers from an install script. Where does CentOS keep its binaries? `/usr/bin/`? How about we copy there instead, if we can't find `/usr/local/bin/` on the host?

Reply:
I evaluated doing that - but several of the service systemd files reference /usr/local/bin.  Some of them I can transformed easily enough, but I wasn't sure about catching all of them - since several of the systemd start up scripts come from - specifically "faasd" and "faas-containerd".

On these systems - /usr/local/bin exists - and in fact is even in root's path.  The problem is explicitly the use of sudo - which has the secure_path which restricts the PATH to exclude /usr/local/bin.